cmake/modules/BaseConfig.cmake

```diff
@@ -132,7 +132,7 @@ endif()
 option(
     TOGGLE_BIN_FOLDER
     "Use build/bin folder for generate compilation files"
-    ON
+    OFF
```

Comment:
is this intentional / part of the change?

Reply:
Yes, it's intentional. It's not necessarily part of the changes. However, it's good for debugging in Visual Studio, as the executable's default workspace is the main folder.